runtime/src/iree/vm/value.h

```diff
@@ -51,6 +53,7 @@ typedef struct iree_vm_value_t {
     int32_t i32;
     int64_t i64;
     float f32;
+    uint16_t f16;
```

Comment:
I wonder if it would make sense to name this `f16_u16`, as the other `union` fields here have a name that conveys the underlying type, potentially making `f16` misleading. @benvanik

Reply:
Ideally, a structured type for `f16 `say `half_t` using `uint16_t` as the storage for the 16 bits. 

```cpp
struct half_t {
   // Storage
  uint16_t data;
  ...
};
```

In the absence of structured type,  `f16_u16` is certainly a better choice here. I will edit it.

Reply:
still TODO